addons/grenades/CfgAmmo.hpp

```diff
@@ -124,4 +124,26 @@ class CfgAmmo {
         soundImpactWoodExt3[] = {"A3\sounds_f\characters\footsteps\wood_run_ext_2",1,1.1,75};
         soundImpactWoodExt4[] = {"A3\sounds_f\characters\footsteps\wood_run_ext_2",1,1.15,75};
     };
+
+    class ACE_G_Incendiary: SmokeShell {
```

Comment:
For consistency with the flashbang this could be renamed `ACE_G_M14`. I'm not sure if it's worth it though.